src/components/BooksForm.js

```diff
@@ -33,25 +33,36 @@ function Form() {
   };
 
   return (
-    <form className="FORM" onSubmit={handleSubmit}>
-      <input
-        type="text"
-        id="title"
-        name="title"
-        required
-        value={state.title}
-        onChange={onChangeHandler}
-      />
-      <input
-        type="text"
-        id="author"
-        name="author"
-        required
-        value={state.author}
-        onChange={onChangeHandler}
-      />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="Add-container">
+      <h2 className="add-title">ADD NEW BOOK</h2>
+      <section>
+        <form className="add-book" onSubmit={handleSubmit}>
+          <input
+            type="text"
+            className="title-input"
+            id="title"
+            placeholder="Add Book Title"
+            name="title"
+            required
+            value={state.title}
+            onChange={onChangeHandler}
+          />
+          <input
+            type="text"
+            id="author"
+            className="author-input"
+            name="author"
+            placeholder="Add Book Author"
+            required
+            value={state.author}
+            onChange={onChangeHandler}
+          />
+          <button className="add-button" type="submit">
+            ADD BOOK
+          </button>
+        </form>
+      </section>
```

Comment:
- [ ] You should stick to the design as much as possible. please use the same border and text color for form. Please compare the design below and make changes 
### Your design
![his](https://github.com/Nuga0909/bookstore3/assets/81727988/344f7c26-3c09-4f68-bf35-ba2ad839420e)

### Original design
![original](https://github.com/Nuga0909/bookstore3/assets/81727988/6c6683fb-59e1-4e16-9df4-af0e33b3f5dc)